ledgerwallet/manifest_toml.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+from typing import Dict, Optional
+
+from ledgerwallet.manifest import AppManifest, icon_from_file
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import toml as tomllib
+
+from ledgerwallet import params
+
+
+class AppManifestToml(AppManifest):
+    def __init__(self, filename):
+        self.path = os.path.dirname(filename)
+        if sys.version_info >= (3, 11):
+            with open(filename, "rb") as f:
+                self.dic = tomllib.load(f)
+        else:
+            with open(filename, "r") as f:
+                self.dic = tomllib.load(f)
+
+    @property
+    def app_name(self) -> str:
+        return self.dic["name"]
+
+    def is_device_defined(self, device) -> bool:
+        if device in self.dic:
+            return "binary" in self.dic[device]
+        return False
+
+    def data_size(self, device) -> int:
+        if "dataSize" not in self.dic[device]:
+            return 0
+        else:
+            return self.dic[device]["dataSize"]
+
+    def get_application_flags(self, device) -> int:
+        if "flags" not in self.dic[device]:
+            return 0
+        else:
+            return int(self.dic[device]["flags"], 16)
+
+    def get_api_level(self, device) -> int:
+        level = self.dic[device]["apiLevel"]
+        if isinstance(level, int):
+            return int(level)
+        return int(level, 10)
+
+    def get_binary(self, device) -> str:
+        return os.path.join(self.path, self.dic[device]["binary"])
+
+    def has_api_level(self, device) -> bool:
+        if device == "0x33100004":
```

Comment:
Could this hard-coded value be stored in a global or an enum easily reachable and understandable?